kogito-quarkus-extension/pom.xml

```
@@ -28,14 +28,34 @@
     </dependencies>
   </dependencyManagement>
 
-  <modules>
-    <module>bom</module>
-    <module>deployment</module>
-    <module>runtime</module>
-    <module>integration-test</module>
-    <module>integration-test-hot-reload</module>
-    <module>integration-test-legacy</module>
-  </modules>
+  <profiles>
+    <profile>
+      <id>default</id>
+      <activation>
+        <activeByDefault>true</activeByDefault>
+      </activation>
+      <modules>
+        <module>bom</module>
+        <module>deployment</module>
+        <module>runtime</module>
+        <module>integration-test</module>
+        <module>integration-test-hot-reload</module>
+        <module>integration-test-legacy</module>
+      </modules>
+    </profile>
+    <profile>
+      <id>prod</id>
+      <activation>
+        <property>
+          <name>prod</name>
```

i would use the same property we use in other assembllies
```suggestion
          <name>productized</name>
```